### DISCUSSION TITLE
DXCC count in V 2.6.16

### DISCUSSION THREAD

**Discussion by Steve-M0SKM** [magicbug/Cloudlog#3200] [Q&A] [answered]
Following install of V2.6.16 I noticed one DXCC less in the count. Georgia is no longer included given the callsign K6VHF/4L. This call was previously valid in Cloudlog and it's recognised and included in LoTW. Any views on this, is it an error?

Steve M0SKM

**Comment by magicbug** [comment]
Odd, nothing has changed to my knowledge when it comes to dxcc code

**Reply by Steve-M0SKM** [reply, marked as answer]
This morning before update the DXCC count was 105; it's now 104 and definitely Georgia that altered. It was a straightforward update which otherwise was great. LoTW still says 105 so I'm not bothered too much.
Thanks for the quick reply Peter.

**Comment by magicbug** [comment]
Will dig into it a bit more this evening see what perhaps changed, guess other thing todo might be run the Update Country Files although that shouldn't really change the numbers!

**Reply by Steve-M0SKM** [reply]
I should have said; I had run Update Country Files already and it could be what provoked it. Got a hospital stay coming up and my mind is elsewhere. Hope you find something.

**Reply by magicbug** [reply]
Thats helpful place to start debugging :)

Good luck with the hospital stay!

**Reply by Steve-M0SKM** [reply]
Just located this; see image 
![cq_zone](https://github.com/user-attachments/assets/267766cb-65f1-4341-97d0-55b839a023a1)
it may help.

**Reply by magicbug** [reply]
Ok that shows the issue the wrong DXCC is selected if you go edit the QSO and set the DXCC to Georgia
